test/calibration_interface.jl

```diff
@@ -0,0 +1,40 @@
+#= Tests for ensuring that the calibration/model interface sets AtmosConfig correctly.
```

Comment:
How long does this take to run? Should it be included in `test/runtests.jl` or on buildkite?

Reply:
This is quick, the only issue is that ClimaCalibrate is unregistered so it needs to be added via `import Pkg; Pkg.add(url="https://github.com/CliMA/ClimaCalibrate.jl.git")` in the test instantiation.

Reply:
Ok, in that case maybe we can add it as soft-fail, until ClimaCalibrate is registered.